extensions/roc-plugin-test-jest/src/config/roc.config.meta.js

```diff
@@ -0,0 +1,29 @@
+import {
+    isPath,
+    notEmpty,
+    isString,
+} from 'roc/validators';
+
+export default {
+    settings: {
+        test: {
+            jest: {
+                __meta: {
+                    description: 'Jest configuration.',
+                },
+                config: {
+                },
+                reporter: {
+                    format: {
+                        description: 'Report format, there is only support for junit',
+                        validator: isString(/junit/),
```

Comment:
Is the idea that more formats will be enabled in the future, since this is a string?